src/components/Directory/MentorCard/MentorCard.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, cleanup, screen } from '@testing-library/react';
```

Comment:
cleanup no longer needed in latest version of RTL, it does it automatically for us behind the scene